pkgs/development/python-modules/openapi-core/default.nix

```
@@ -1,85 +1,80 @@
 { lib
-, attrs
 , buildPythonPackage
-, dictpath
 , django
 , djangorestframework
 , falcon
 , fetchFromGitHub
+, fetchpatch
 , flask
 , isodate
-, lazy-object-proxy
-, mock
+, jsonschema-spec
 , more-itertools
 , openapi-schema-validator
 , openapi-spec-validator
 , parse
+, pathable
+, poetry-core
 , pytestCheckHook
 , pythonOlder
 , responses
-, six
+, typing-extensions
 , webob
 , werkzeug
-, python
 }:
 
 buildPythonPackage rec {
   pname = "openapi-core";
-  version = "0.14.2";
-  format = "setuptools";
+  version = "0.15.0";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "p1c2u";
     repo = "openapi-core";
     rev = version;
-    hash = "sha256-+VyNPSq7S1Oz4eGf+jaeRTx0lZ8pUA+G+KZ/5PyK+to=";
+    hash = "sha256-7RjKll+z9wrax9hLS4cKpQLOvmlD+lXbk9CkZi0oSbw=";
   };
 
```

```suggestion

  postPatch = ''
    substituteInPlace pyproject.toml \
      --replace "--cov=openapi_core" "" \
      --replace "--cov-report=term-missing" "" \
      --replace "--cov-report=xml" ""
  '';

```
This is not "fixing an error" but removing arguments for pytest plugins we purposely do not want to add.